rwr.py

```diff
@@ -260,8 +260,8 @@ class PlayersSort:
     XP = 'rank_progression'
 
 
-class PlayersListGame:
-    VANILLA = 'invasion'
+class PlayersListDatabase:
+    INVASION = 'invasion'
     PACIFIC = 'pacific'
```

Comment:
Refactor this class to a e.g dict and create appropriate functions (lmike server's type and mode)